### ISSUE TITLE
This library is just generating random strings not the words from english dictionary

### ISSUE THREAD

**Issue by pradeep-orbi** [maximeroussy/invitrode#6]
This library is just generating random strings not the words from english dictionary

**Comment by maximeroussy** [comment]
That's correct. The library isn't meant to produce real words from the dictionary, but they're not entirely random strings. 

The key part of the description is "english pronounceable", meaning it constructs words based on rules of the english language to ensure you can pronounce them. But, unless it randomly creates a real word by chance, most of the words generated will be fictitious.
